compiler/rustc_lexer/src/lib.rs

```diff
@@ -27,11 +27,13 @@
 // tidy-alphabetical-end
 
 mod cursor;
-pub mod unescape;
 
 #[cfg(test)]
 mod tests;
 
+// FIXME: This is needed for rust-analyzer. Remove this dependency once rust-analyzer uses
+// `literal-escaper`.
```

Comment:
This is a sub-tree so you can just make the changes directly.

Reply:
For now it's tricky because it requires to make `literal_escaper` a dependency of r-a. I'll check with r-a people directly after this is merged.

Reply:
This will break our auto publishing of `rustc_lexer` and I don't immediately see how to prevent that :/ That means we won't be able to sync our subtree after this change

Reply:
No wait, this will break `rustc_lexer` and `rustc_parse_format_args` in general if I see this right? Those crates need to compile on stable which they no longer will by depending on this new unstable standard library crate.

Reply:
Yeah, so for the time being, I leave it as is. Let's come up with a solution later on. :)